src/styles.scss

```diff
@@ -1,5 +1,52 @@
 /* You can add global styles to this file, and also import other style files */
-html, body {
-	height: 100%;
+
+@import '~variables.scss';
+html,
+body {
+  height: 100%;
+}
+
+/* delete dialog custom styles */
+.modal .modal-dialog {
```

Comment:
styles of delete-dialog modal should be part of delete-dialog component

Reply:
two modal windows use this styles

Reply:
if i move them to delete-dialog scss file they don't work , i can assume that it can be related that delete dialog is created on global level out of our main component, if i set ViewEncapsulation.None for component it will start working. These styles are being used by Vadim as well, does it make sense to leave them where they are and rename delete-dialog selector to something more common ?

Reply:
Olga, as I mentioned it in our chat all component-specific styles should go into the component's scss file. If it is common styles for all modals you can keep in in styles, and renaming it is a good option